langchain/__init__.py

```diff
@@ -15,9 +15,11 @@
     SQLDatabaseChain,
 )
 from langchain.docstore import Wikipedia
+from langchain.example_generator import generate_example  # noqa
```

Comment:
if we're not going to put them in __all__ (which i think is right), lets not import them here